09.spec.js

```diff
@@ -0,0 +1,16 @@
+let MainPage = require("../pages/main.page");
```

Comment:
Looks like extra file

Reply:
I'm clicking on Feedback link from Main page, this file is required

Reply:
I mean `09.spec.js` there is same in specs folder